app/helpers/katello/concerns/settings_helper_extensions.rb

```diff
@@ -12,6 +12,8 @@ def value(setting)
             'katello_default_provision',
             'katello_default_ptable',
             'katello_default_PXELinux',
+            'katello_default_PXEGrub',
+            'katello_default_PXEGrub2',
```

Comment:
00:04:49.998 /var/lib/workspace/workspace/test_katello_pull_request/slave/fast/plugin/app/helpers/katello/concerns/settings_helper_extensions.rb:7:9: C: Metrics/MethodLength: Method has too many lines. [35/30]
00:04:49.998         def value(setting) ...

I think this needs a rubocop override

Reply:
Do you prefer local or global?

Reply:
Amended local comment disable.

Reply:
Yep, local. Thanks! 👍